media/wav.ksy

```diff
@@ -0,0 +1,272 @@
+meta:
+  id: wav
+  file-extension: wav
+  title: Microsoft WAVE audio file
+  endian: le
+  license: BSD-3-Clause-Attribution
+  encoding: ASCII
+doc: |
+  The WAVE file format is a subset of Microsoft's RIFF specification for the
+  storage of multimedia files. A RIFF file starts out with a file header
+  followed by a sequence of data chunks. A WAVE file is often just a RIFF
+  file with a single "WAVE" chunk which consists of two sub-chunks --
+  a "fmt " chunk specifying the data format and a "data" chunk containing
+  the actual sample data.
+  This Kaitai implementation was written by John Byrd of Gigantic Software
+  (jbyrd@giganticsoftware.com), and it is likely to contain bugs.
+doc-ref: http://soundfile.sapp.org/doc/WaveFormat/
+doc-ref: https://www.loc.gov/preservation/digital/formats/fdd/fdd000001.shtml
+seq:
+  - id: riff_id
+    contents: RIFF
+  - id: file_size
+    type: u4
+  - id: wave_id
+    contents: WAVE
+  - id: chunks
+    type: chunks_type
+    size: file_size - 5
+  - id: unused
+    size-eos: true
```

Comment:
This is not really necessary to grab the end of file as a distinct attribute. Actually, if anyone wants it, it will have the file pointer positioned right here after finishing the parsing, so it's easy to read it (or invoke another parser, etc), I guess?